src/constants/externalResources.ts

```diff
@@ -19,3 +19,18 @@ export const getJiraSearchUrl = (jiraHost: string, jqlEscaped: string) =>
 
 export const getJiraTicketUrl = (jiraHost: string, jiraKey: string) =>
   `https://${jiraHost}/browse/${jiraKey}`;
+
+export const getLobsterTestLogUrl = (
+  taskId: string,
+  execution: number,
+  testId: string,
+  groupID?: string,
+  lineNum?: number
+) =>
+  !taskId || Number.isNaN(execution) || !testId
+    ? ""
+    : `${getLobsterURL()}/lobster/evergreen/test/${taskId}/${execution}/${testId}${
+        groupID ? `/${groupID}` : ""
+      }${lineNum ? `#bookmarks=${lineNum}` : ""}`;
+
```

Comment:
I think this function might be producing bad links. When connected to prod try opening a test log with the lobster button with this task
http://localhost:3000/task/mongodb_mongo_master_enterprise_rhel_80_64_bit_dynamic_all_feature_flags_required_display_replica_sets_ese_gcm_patch_38df2a19a17bd90c70ee0a490afb824977b655c8_605a51821e2d1753ad64313f_21_03_23_20_37_46/tests?execution=0

The [link](https://logkeeper.mongodb.org/lobster/evergreen/test/mongodb_mongo_master_enterprise_rhel_80_64_bit_dynamic_all_feature_flags_required_display_replica_sets_ese_gcm_patch_38df2a19a17bd90c70ee0a490afb824977b655c8_605a51821e2d1753ad64313f_21_03_23_20_37_46/0/job0_fixture_setup_0#l=1) thats produced from this function does not work . Lobster never loads anything

Clicking on the html button [link](https://logkeeper.mongodb.org/lobster/build/5309e0a976c4fac365215b49aa235e8b/test/605a5f96be07c47613c83c07#bookmarks=0%2C3&l=1) does work correctly though

Reply:
you should export REACT_APP_LOBSTER_URL=https://evergreen.mongodb.com from .cmdrc.json
I updated the isLobsterLink function so the conditional render should make more sense. 
checkout this page which only has two log buttons (Lobster, Raw) as opposed to 3 (Lobster, HTML, Raw). We show only 2 buttons when the HTMLDisplayURL field for the testlog point to lobster instead of the evergreen html viewer. 

Check out this page after pulling the latest commits to see how we render only 2 buttons: http://localhost:3000/task/mongodb_mongo_master_enterprise_rhel_80_64_bit_dynamic_required_aggregation_disabled_optimization_patch_e9f91c18323784288cd89cbbb70bc0eca8e89aae_602ff95457e85a5721939bc6_21_02_19_17_46_09/tests?execution=0&sortBy=STATUS&sortDir=ASC